### PR TITLE
Modeling Reinforced Concrete Walls with Shell Elements to Simulate Through Opensees and Using Jupyter to Post Process Results

### PR DESCRIPTION
**Pull request by stokljos**
Testing

### CODE REVIEW

File: docs/lowes/usecase.md [hunk 1]
```diff
@@ -1,89 +1,134 @@
...
+[Jupyter Notebook on DesignSafe](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis)  
+[Opensees on DesignSafe](https://www.designsafe-ci.org/rw/workspace/#!/OpenSees::Simulation)  
 
-## Background 
+
+<!--- test commment --->    
+<!--- this is a comment --->  
+<!--- this is a comment --->  
+<!--- this is a comment --->  
+<!--- this is a comment --->
```
Comment (kks32) [inline docs/lowes/usecase.md]:
```suggestion
```

File: docs/lowes/usecase.md [hunk 2]
```diff
@@ -1,89 +1,134 @@
...
 ### Citation and Licensing
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+* Please cite [Shegay et al. (2021)](https://doi.org/10.17603/ds2-r12q-t415) to acknowledge the use of any data from this use case.
+
+* Please cite [Lu XZ et al. (2015)](http://www.luxinzheng.net/download/OpenSEES/En_THUShell_OpenSEES.htm ) to acknowledge the use of the modeling strategy from this use case.
 
 * Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).
```
Comment (kks32) [inline docs/lowes/usecase.md]:
```suggestion
* This software is distributed under the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.html).  
```

File: docs/lowes/usecase.md [hunk 3]
```diff
@@ -1,89 +1,134 @@
...
+
+* PlaneStressUserMaterial- Utilizes damage mechanisms and smeared crack model to defin a multi-dimensional concrete model  
+    * Variables include: compressive strength, tensile strength, crushing strength, strain at maximum and crushing strengths, ultimate tensile strain, and shear retention factor
+    * Model can be found in Lu XZs citation  
+ 
+* Steel02- Uniaxial steel material model with isotropic strain hardening
+    * Variables include: yield strength, initial elastic tangent, and strain hardening ratio  
+    * Model can be found here: [Steel02 OpenSees](https://opensees.berkeley.edu/wiki/index.php/Steel02_Material_--_Giuffr%C3%A9-Menegotto-Pinto_Model_with_Isotropic_Strain_Hardening)  
+
+<img src="img/ShellEle.JPG" width="500" height="250" />
```
Comment (kks32) [inline docs/lowes/usecase.md]:
```suggestion
<img src="../img/ShellEle.JPG" width="500" height="250" />
```

File: docs/lowes/usecase.md [hunk 4]
```diff
@@ -1,89 +1,134 @@
...
-*Add images to the folder img and use relative path to specify the location of the image.*   
 
-![caption](img/mkdocs-template.png)
-> Use case template design
 
+### Modeling Script 
 
-## Header3
+The sections of the modeling script are: [Modeling Script](https://jupyter.designsafe-ci.org/user/stokljos/notebooks/MyData/UseCaseScripts/TCL_Script_Creator.ipynb)  
+* Section 1: Initializion of the model.
```
Comment (kks32) [inline docs/lowes/usecase.md]:
```suggestion

* Section 1: Initializion of the model.  
```

File: docs/lowes/usecase.md [hunk 5]
```diff
@@ -1,89 +1,134 @@
...
-## Header3
+The sections of the modeling script are: [Modeling Script](https://jupyter.designsafe-ci.org/user/stokljos/notebooks/MyData/UseCaseScripts/TCL_Script_Creator.ipynb)  
+* Section 1: Initializion of the model.  
+   * The degrees of freedom and the variables that carry uncertainty are defined.  
+* Section 2: Defines nodal locations and elements.  
+   * Nodes are placed at the locations of the vertical bars along the length of the wall
+   * If the ratio of the length of the wall to the number of elements is too coarse of a mesh, additional nodes are placed inbetween the bars.
+   * The height of each element is equal to the length of the nodes in the boundary to create square elements up the wall.
+* Section 3: Defines material models and their variables.
+   * The crushing energy and fracture energy are calculated and wrote to the .tcl file. The equations for these values come from (Nasser et al.)  Below is the code:
```
Comment (kks32) [inline docs/lowes/usecase.md]:
```suggestion
   * The crushing energy and fracture energy are calculated and wrote to the .tcl file. The equations for these values come from (Nasser et al.)  Below is the code:
   
```

File: docs/lowes/usecase.md [hunk 6]
```diff
@@ -1,89 +1,134 @@
...
 
-*Add images to the folder img and use relative path to specify the location of the image.*   
 
-![caption](img/mkdocs-template.png)
-> Use case template design
 
+### Modeling Script 
 
-## Header3
+The sections of the modeling script are: [Modeling Script](https://jupyter.designsafe-ci.org/user/stokljos/notebooks/MyData/UseCaseScripts/TCL_Script_Creator.ipynb)
```
Comment (kks32) [inline docs/lowes/usecase.md]: This script is not in Community Data and cannot be viewed by others. Please update the link.

File: docs/lowes/usecase.md [hunk 7]
```diff
@@ -1,89 +1,134 @@
...
-| Column 1 | Column 2 | Column 3 |
-|----------|----------|----------|
-| Stampede2| CPU      | 2017     |     
-| Frontera | CPU & GPU| 2019     |     
+## Post Processing
 
-Or use markdown table generator: [https://www.tablesgenerator.com/markdown_tables](https://www.tablesgenerator.com/markdown_tables)
+After the script is finished running through OpenSees, there are multiple post-processing scripts that can be used to analyize the simulation and compare it to the experimental numbers.
 
+### Load-Displacment Graph
```
Comment (kks32) [inline docs/lowes/usecase.md]:
```suggestion
### Load-Displacement Graph
```

**Review by kks32 — CHANGES_REQUESTED** [review]
Please make suggested changes

**Review by kks32 — CHANGES_REQUESTED** [review]
I cannot access the Jupyter Notebooks, please make them available in the community data so I can merge.

**Comment by stokljos** [comment]
I will update that soon and let you know when it is done!

**Comment by stokljos** [comment]
The links are updated. Let me know if there's anything else I can do for
you.

**Review by kks32 — APPROVED** [review]

**Pull request merged**